test/samlp.tests.js

```diff
@@ -571,4 +571,93 @@ describe('samlp', function () {
       });                  
     });
   });
+
+  describe('configured signature prefix', function(){
```

Comment:
Can you add a test when there is no prefix? (maybe it is worthless if other tests cover that)